### ISSUE TITLE
Cannot batch upload PDFs

### ISSUE THREAD

**Issue by augustincaitlin** [datakind/permanent-colandr-back#28]
Users currently have to upload pdfs one by one.

**Comment by bdewilde** [comment]
I agree that this is inconvenient, but I'm not sure how else we can know how a particular PDF in a batch goes with a particular study to be reviewed. Open to suggestions!
